Installing-and-Configuring/OS/index.md

```diff
@@ -0,0 +1,136 @@
+
+If the disk is moved to a different system by the owner the LUKS Disk Recovery Key passphrase is used to unlock encrypted container from slot 0 to access the encrypted data.  This model is similar to the [Security Dongle](#security-dongle-with-luks-root) model.
+
+Generally, an attacker would not have the LUKS Disk Unlock key unlocking key slot 1, because it was generated under Heads with a random 128 characters key once and then sealed into TPM NV memory space, to be unsealed only when provided with proper measurements and Disk Unlock Key passphrase. To be able to access this content in TPM, or observe that secret traveling from TPM to memory, physical access would be required with the help of a TPM Genie or other physical attack combinations.
+
+The attacker should neither have access to the LUKS Disk Recovery Key (LUKS keyslot 0) nor its associated passphrase, because it was setuped at install time and should only be typed from the user when renewewing/changing the Disk Unlock Key passphrase when setting a new boot default after Heads detects changed core /boot components and prompts user to detach sign /boot digest (sha256sum).
+
+Even if the attacker clones the drive, he would still not have access to the decrypted data, unless he eavesdropped the Disk Recovery Key passphrase which was prompted when defining a new boot default, which should be done right after having upgraded QubesOS dom0, from a safe place where eavesdropping is highly improbable.
+
+To mitigate recovery shell attacks, entering the recovery shell will modify the PCR and measurements. Also changes to kernel or initrd files on /boot will modify the measurements.  This will be detected as tampering.
+
+See also [TPM vs security dongle](#tpm-vs-security-dongle)
+
+#### Disable TPM Disk Unlock passphrase/key
+
+This feature may be turned off during configuration of heads.  The following will instruct heads not to ask for a disk unlock passphrase and to not store a disk unlock key in the TPM.  
+
+  CONFIG_TPM_NO_LUKS_DISK_UNLOCK=y
+
+### Using Heads Disk Recovery Key 
+
+In the Disk Recovery Key scenario, Heads doesn't attempt to release the Disk Unlock Key. It simply relies on OS mechanisms to directly prompt the user for the Disk Recovery Key passphrase at standard OS prompt.  
+
+This model is vulnerable to brute force attacks.
+
+### Using an OS managed LUKS passphrase (not heads)
+
+In this model heads is not involved in processes related to the decryption of data on disk. The OS manages keys and access to the LUKS containers by asking the user for a passphrase on boot.  
```

Comment:
Those two are the same. If Heads is not releasing the Disk Unlock Key from TPM, then it asks for the Disk Recovery Key, defined at OS install (Which is factually the Disk Recovery Key passphrase or LUKS decryption key)